src/test/harness.rs

```diff
@@ -212,12 +219,9 @@ pub struct Cache {
 }
 
 impl Cache {
-    fn new(base_path: &Path) -> Cache {
-        let mut root_path = env::current_dir().expect("Could not find current working directory");
-        root_path.push(base_path);
-
+    fn new(base_path: PathBuf) -> Cache {
         Cache {
-            base_path: root_path,
+            base_path: base_path,
```

Comment:
Could just use `base_path` here instead of `base_path: base_path`

Reply:
Done.